packages/atomic/src/components/result-template-components/atomic-result-badge/atomic-result-badge.pcss

```diff
@@ -5,3 +5,7 @@
   place-items: center;
   height: 100%;
 }
+
+.result-badge-element {
+  margin-right: 8px;
```

Comment:
> I did not find anything in tailwind documentation to allow to specify fixed 8px

Could it be due to another style not using tailwind, that if adjusted to use the tailwind design system, would remove the need for custom CSS?